README_MULTIVAC.md

```diff
@@ -0,0 +1,77 @@
+research abstracts to the QG-Net context reader. The abstract is the section
+most authors use to communicate the research succinctly while highlighting its
+most important facets. MULTIVAC uses the five most important sentences in the
+abstract as the "context" and uses the most important words/phrases in each of
+the five sentences of the abstract as the “answer.” Term frequency-inverse
+document frequency (TF-IDF) scores are calculated to determine which terms/
+sentences in the documents are truly important and differentiating. The fives
+sentences with the largest total sum of their terms' TF-IDF scores are used to
+generate queries. Within the top five sentences, the term or consecutive terms
+with the highest TF-IDF scores are tagged as "answers."
+
+![alt text](https://github.com/GallupGovt/multivac/blob/master/images/qgnet.png 'QG-net schematic')
+<br>Illustration of the QG-Net system. Source: https://github.com/moonlightlane/QG-Net
+
+Given context and answer inputs, QG-Net generates different questions that
+focus on the relevant contextual information that different answers provide.
+More specifically, QG-Net uses [GloVe](https://nlp.stanford.edu/projects/glove/),
+an unsupervised learning algorithm for modeling word embeddings, to represent
+words as vectors coupled with speech tag (POS), name entity recognition (NER),
+and word case (CAS) from the Stanford natural language processing toolkit as
+inputs in the context reader. In addition, the QG-Net input includes a
+binary-valued indicator to indicate whether a word is the "answer." Finally,
+the question generator generates diverse question text word-by-word given all
+context word representations.
+
+## Query Output
+The resulting queries are then applied in order to MULTIVAC’s Markov logic
+network (MLN) ontology and the resulting answers are compared with the "answer"
+extracted directly from the source data. Good matches, in the sense of high
+scores for semantic completeness, lexical similarity, and syntactical
+similarity will indicate good performance. These measures will be assessed
+using approaches including but not necessarily limited to latent semantic
+analysis and minimal edit distance metrics. With QG-Net, we generated 2,804
+queries based on epidemiology research abstracts scraped and processed as
+earlier steps in MULTIVAC.
+
+## Dependencies
+To run QG-Net, there are a number of dependencies above and beyond those required for MULTIVAC. Primary among them is the need for a GPU-based system (currently tested against a Nvidia Quadro Pro-4000 card) for computation of the RNN. In addition to the GPU, the following are core requirements:
+- GPU for computation
+- Python 3.6
+- pytorch-0.4.1
+- OpenNMT-py (note, the version of OpenNMT-py in the original QG-Net is necessary for execution; not tested with MIT's version of OpenNMT)
+- [Stanford CoreNLP server](https://stanfordnlp.github.io/CoreNLP/#download) (optional)
```

Comment:
The Stanford CoreNLP server is NOT optional. I think the original author said it was optional because he already generated sample input text for the QG-Net model. However, to generate input for the QG-Net model you must use the Stanford CoreNLP server.

Reply:
Okay, good to know. This will require some fixes elsewhere, but thank you for the heads up. Part of the reason I went with 'optional' was due to it being commented out in some of your code. No biggie -- will add it back in.